presto-docs/src/main/sphinx/admin/session-property-managers.rst

```diff
@@ -69,20 +69,20 @@ These requirements can be expressed with the following rules:
     [
       {
         "group": "global.*",
-        "sessionProperties": {
+        "session_properties": {
```

Comment:
Is this a breaking change or do you have code to pull in the old name as well?

Reply:
The old name is still supported via `@JsonAlias`.

Reply:
Great .. not sure if we need to even add this as a note to the release notes then.